### PR TITLE
Added LZ4 compression

### PR DESCRIPTION
**Pull request by railsmechanic**
Added LZ4 compression...

### CODE REVIEW

File: tarlz4.go [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+type tarLz4Format struct{}
+
+func (tarLz4Format) Match(filename string) bool {
+	return strings.HasSuffix(strings.ToLower(filename), ".tar.lz4") || strings.HasSuffix(strings.ToLower(filename), ".tlz4") || isTarLz4(filename)
+}
+
+// isTarLz4 checks the file has the bzip2 compressed Tar format header by
+// reading its beginning block.
+func isTarLz4(tarlz4Path string) bool {
```
Comment (mholt) [inline tarlz4.go]: The comment says bzip2, but the code just checks the tar format?

File: tarlz4.go [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+	return hasTarHeader(buf)
+}
+
+// Make creates a .tar.bz2 file at tarlz4Path containing
+// the contents of files listed in filePaths. File paths
+// can be those of regular files or directories. Regular
+// files are stored at the 'root' of the archive, and
+// directories are recursively added.
+func (tarLz4Format) Make(tarlz4Path string, filePaths []string) error {
```
Comment (mholt) [inline tarlz4.go]: Don't forget to update the godoc comments ;)

**Comment by mholt** [comment]
Why did you close this one?

**Comment by railsmechanic** [comment]
Sorry my fault... Plz ignore this one and head over to PR #24 

Many thanks 👍

**Pull request closed without merging**